src/commands/CommandHandler.ts

```diff
@@ -170,7 +176,11 @@ export async function handleCommand(roomId: string, event: { content: { body: st
                 "!mjolnir shutdown room <room alias/ID> [message]                    - Uses the bot's account to shut down a room, preventing access to the room on this server\n" +
                 "!mjolnir powerlevel <user ID> <power level> [room alias/ID]         - Sets the power level of the user in the specified room (or all protected rooms)\n" +
                 "!mjolnir make admin <room alias> [user alias/ID]                    - Make the specified user or the bot itself admin of the room\n" +
-                "!mjolnir help                                                       - This menu\n";
+                "!mjolnir suspend <user ID>                                         - Suspend the specified user"
+                 +
+                "!mjolnir unsuspend <user ID>                                        - Unsuspend the specified user"
+                +
+                "!mjolnir help                                                       - This menu\n"
```

Comment:
```suggestion
                "!mjolnir suspend <user ID>                                          - Suspend the specified user" +
                "!mjolnir unsuspend <user ID>                                        - Unsuspend the specified user" +
                "!mjolnir help                                                       - This menu\n";
```

I'm a bit surprised the linter didn't catch this.